### PR TITLE
Added requirements.txt and update the readme to include it

### PR DESCRIPTION
**Pull request by tmcdonal**

### CODE REVIEW

**Review by PoesRaven — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,21 +30,25 @@ QChat leverages event invokation to broadcast realtime messages across the chann
 
 ## Setup
 
-### McAfee OpenDXL SDK
+### Dependencies
 
-https://www.mcafee.com/us/developers/open-dxl/index.aspx
+Qchat only supports Linux Desktop systems.
```
Comment (PoesRaven) [inline README.md]: QChat uses appJar which is platform independent. I've tested with Linux, Mac, and Windows.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # OpenDXL-QChat
-QChat (Quick Chat) is a chat room service leveraging the OpenDXL event invokation capabilities to create small, light-weight, and interactive chat rooms for use by incident responders and SOC personnel.
+QChat (Quick Chat) is a chat room service for Linux Desktop systems leveraging the OpenDXL event invokation capabilities to create small, light-weight, and interactive chat rooms for use by incident responders and SOC personnel.
```
Comment (PoesRaven) [inline README.md]: This has been tested with Linux, Windows, and Mac.

**Review by PoesRaven — CHANGES_REQUESTED** [review]
QChat uses appJar which is platform independent. I've tested with Windows, Linux, and Mac.

File: README.md (hunk 1 above)
Comment (tmcdonal) [reply README.md]: Ah, sorry, the dependency on curses is why I thought it was only supporting Linux. What library are you using in Windows to satisfy the curses dependency?

Comment (PoesRaven) [reply README.md]: The curses import has been removed and code updated to reflect. It was antiquated pre appJar implementation.

**Review by PoesRaven — CHANGES_REQUESTED** [review]
Please update te reflect platform before merging.

**Comment by tmcdonal** [comment]
I've updated the documentation to remove the references to Linux.

**Pull request merged**